src/schemas/21-526EZ/schema.js

```diff
@@ -14,7 +14,7 @@ const disabilitiesBaseDef = {
     required: ['name', 'disabilityActionType'],
     properties: {
       name: {
-        type: 'string',
+        type: 'string'
```

Comment:
This file only changed because I stupidly made a noop change and saved it, which triggered Prettier to reformat the whole thing. In any case, I'm going to keep this new formatting for consistency, and because it didn't change the build file anyway, so no impact to the actual form.